release-content/0.14/migration-guides/12314_Clean_up_type_registrations.md

```diff
@@ -1,3 +0,0 @@
-External types are no longer registered automatically unless they are used by
```

Comment:
This was added in this PR: https://github.com/bevyengine/bevy-website/pull/1388 after it was discovered to be more breaking than previously assumed.

Reply:
See also discussion starting around here: https://discord.com/channels/691052431525675048/1239930965267054623/1250181057584824451 since that PR doesn't include a description.

Reply:
Got it!